app/models/deleted_project.rb

```diff
@@ -0,0 +1,15 @@
+class DeletedProject < ApplicationRecord
+  scope :updated_within, ->(start, stop) { where('updated_at >= ? and updated_at <= ? ', start, stop).order(updated_at: :asc) }
```

Comment:
Other question is: why not just mark a "deleted_at" date on the project? Just too many things that can potentially still be using at to play whack-a-mole with?